spec/import_spec.rb

```diff
@@ -112,21 +132,30 @@ def upload_file!(name, ext='csv')
         end
       end
 
-      context "Win1251" do
-
-        before do
-          upload_file!(:authors_win1251_win_endline)
-        end
+      context "auto detect encoding" do
+        include_examples 'successful inserts' , :auto,  :authors_win1251_win_endline
+      end
 
-        it "should import file with many records" do
-          expect(page).to have_content "Successfully imported 2 authors"
-          expect(Author.count).to eq(2)
-          Author.all.each do |author|
-            expect(author).to be_valid
-            expect(author.name).to be_present
-            expect(author.last_name).to be_present
-          end
-        end
+      context "Win1251" do
+        include_examples 'successful inserts' , 'windows-1251',  :authors_win1251_win_endline
```

Comment:
Line is too long. [93/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.